spec/models/work_spec.rb

```diff
@@ -604,6 +604,20 @@
 
         expect(draft_work.curator).to eq(curator_user)
       end
+
+      it "sends a notification for the approval of the work" do
+        allow(WorkActivity).to receive(:add_system_activity)
+        draft_work = FactoryBot.create(:draft_work)
+        file_name = uploaded_file.original_filename
+        stub_work_s3_requests(work: draft_work, file_name: file_name)
+        draft_work.pre_curation_uploads.attach(uploaded_file)
+
+        draft_work.complete_submission!(user)
+        draft_work.update_curator(curator_user.id, user)
+        draft_work.approve!(curator_user)
+
+        expect(WorkActivity).to have_received(:add_system_activity).with(draft_work.id, "successfully published", curator_user.id)
```

Comment:
I think the original issue sets a higher bar for testing:
> There is an automated test that checks that these notifications get sent

I think this is checking that the right method is getting called, but I don't think it's checking that the email actually gets sent.

Maybe there are other tests that already cover that? I wouldn't block if this seems sufficient to other folks.

Reply:
I am very sorry for this, I will extend the test suite in order to ensure that the e-mail messages are sent.

Reply:
My memory of the conversation is that Bess agreed that this is sufficient, and Carolyne's work will handle the end-to-end process of sending a message, but whatever you think best.

Reply:
I apologize, due to being unwell, and wishing to advance this, I will prepare this for review and merge please.